### ISSUE TITLE
console.log statement being added to wrong line during array destructuring.

### ISSUE THREAD

**Issue by phiter** [Chakroun-Anas/turbo-console-log#121]
If I run the command on a object declaration variable, the console.log statement is added after closing the object.
But in this specific use case, if I pick the destructured variable of a call to a function that takes an array, the statement is being added to the wrong line:

![Peek 2021-02-15 08-59](https://user-images.githubusercontent.com/12252332/107944079-5d748300-6f6c-11eb-8688-e60ea341a6a5.gif)

Code:
```ts
const [tags, setTags] = useState<Tag[]>([
    { id: 1, name: 'Apple' },
    { id: 2, name: 'Pear' }
]);
```

**Comment by Chakroun-Anas** [comment]
Hello,

Thank you for sharing this use case, i just pushed a fix `v2.3.1` can you please check it and get back to me so that we can close this issue? thx.

**Comment by Chakroun-Anas** [comment]
Not relevant anymore.
